float64 x
float64 y
float64[4] covariance
int16 id
string cls
